File: Ej_09-03-26_While.cpp
//While - Mientras

#include <iostream>
using namespace std;

int main() {
	//Declarar variables
	//Inicializadas
	int mdo = 8, mdor = 0, prod;

	cout << "Tablas" << endl;
	cout << "\nIngresa un numero: "; cin >> mdo;

	while (mdor <= 9) {
		mdor++;	//mdor = mdor + 1; //contador
		prod = mdo * mdor;

		//Datos de salida
		cout << mdo << " * " << mdor << " = " << prod << endl;
		//Fin Mientras
	}
	return 0;
} //Fin main

File: Ej_09_03_26_While.py
#While - Mientras

mdo = 8 
mdor = 0

print("Tablas\n")
mdo = int(input("Ingresa un numero: "))

while mdor <= 9:
    mdor += 1    #mdor = mdor + 1
    prod = mdo * mdor

    print(f"{mdo} * {mdor} = {prod}")
    #print(mdo, "*", mdor,"=",prod)
